src/factory/README.md

```diff
@@ -32,7 +34,33 @@ You can check the factory file to see more details, it consists of 4 methods, de
 }
 ```
 
-### 2. Calculate new module address
+### 2. Deploy and set up an unknown module
```

Comment:
Perhaps we could change this to `deployAndSetUpCustomModule()`, to keep the language consistent with the Zodiac app.